src/backend/base/langflow/graph/edge/schema.py

```diff
@@ -32,3 +34,55 @@ def format(self, sep: str = "\n") -> str:
                 for result_pair in self.result_pairs[:-1]
             ]
         )
+
+
+class TargetHandle(BaseModel):
+    fieldName: str = Field(..., description="Field name for the target handle.")
```

Comment:
`field_name: str = Field(..., alias="fieldName", description="Field name for the target handle.")`